x-pack/plugins/observability/server/ui_settings.ts

```diff
@@ -251,6 +252,24 @@ export const uiSettings: Record<string, UiSettings> = {
     ),
     schema: schema.boolean(),
   },
+  [enableInfrastructureHostsCustomDashboards]: {
+    category: [observabilityFeatureId],
+    name: i18n.translate('xpack.observability.enableInfrastructureHostsCustomDashboards', {
+      defaultMessage: 'Custom dashboards for Host Details in Infrastructure',
+    }),
+    value: false,
+    description: i18n.translate(
+      'xpack.observability.enableInfrastructureHostsCustomDashboardsDescription',
+      {
+        defaultMessage:
+          '{betaLabel} Enable option to link custom dashboards in the Host Details view.',
```

Comment:
QQ: what does `to link` mean in this context?

Reply:
The final UI will say to "link" a dashboard to a an asset, so I've decided to stick to that terminology here as well.
![CleanShot 2024-02-05 at 10 41 36@2x](https://github.com/elastic/kibana/assets/793851/22bbb5c4-5006-4cb1-9b39-aef656fe649c)